valhalla/jawn/src/lib/shared/db/s3Client.ts

```diff
@@ -0,0 +1,56 @@
+import { S3Client as AwsS3Client, GetObjectCommand } from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import { Result } from "../result";
+
+export type RequestResponseBody = {
+  request?: any;
+  response?: any;
+};
+
+export class S3Client {
+  private region = "us-west-2";
+  private awsClient: AwsS3Client;
+
+  constructor(
+    accessKey: string = "minioadmin",
+    secretKey: string = "minioadmin",
+    endpoint: string = "http://localhost:9000",
```

Comment:
umm are we sure we ant to default here? ... I feel like we should throw if it is not explicitly defined in the environment

Reply:
fixed